infra/examples-dev/aws-msft-365/main.tf

```diff
@@ -8,7 +8,7 @@ terraform {
 
     # for API connections to Microsoft 365
     azuread = {
-      version = "~> 2.0"
+      version = "~> 2.3"
```

Comment:
Need to bump it because otherwise `audience` will require to be a valid URI when creating a federated credential; true for GCP but not for AWS.

Reply:
OK, add note in `CHANGELOG` that customers will need to `terraform init --upgrade` if coming from older version.